client/src/components/App.jsx

```diff
@@ -55,6 +60,12 @@ class App extends React.Component {
         <h2>
           <p> Price: ${this.state.displayProduct.default_price}</p>
         </h2>
+        <br></br>
```

Comment:
Andrew - 

You may consider putting your <h3> tag  inside of your widget component. This is just a suggestion to keep our  main app component as clean as possible.  

Note:   lines 55-62 will definitely be removed in future merges.

Reply:
<br> removed ))